src/screens/DiscoveryScreenV2/hooks/useDiscovery.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 
+import {uniqBy} from 'lodash';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `uniqBy` from lodash is a good move to ensure uniqueness in the array. However, it's important to note that lodash functions can sometimes be heavy and impact performance. If the `discovery.initialTopics` array is not expected to have a large number of elements, this should be fine. Otherwise, consider implementing a lighter custom function for uniqueness check.
```diff
+ import {uniqBy} from 'lodash';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->